easyprivacy/easyprivacy_specific_international.txt

```diff
@@ -309,6 +310,7 @@
 ||statistic.takungpao.com^
 ||stats.autohome.com.cn^
 ||stats.yinyuetai.com^
+||sugar.zhihu.com^
```

Comment:
Touches https://forums.lanik.us/viewtopic.php?p=151451#p151451

Reply:
Hello! The file seems to have been renamed now and disabling the domain does not seem to have any effect

Reply:
Current tracking domain is `zhihu-web-analytics.zhihu.com` (https://mypdns.org/my-privacy-dns/matrix/-/issues/1176) but it is still resolving.

Reply:
Ok, when I have time I will open another Pull Request and delete this domain, thank you!

Reply:
> Ok, when I have time I will open another Pull Request and delete this domain, thank you!

You can just edit the file you have submitted and the re-submit it

Reply:
Okay
Then please ask
```
analytics.zhihu.com
zhihu-analytics.zhihu.com
```
Do I need to add these two domains? Thank you

Reply:
Ping @ryanbr @Khrin :arrow_double_up:

Reply:
```
||unpkg.zhimg.com/za-js-sdk^
||unpkg.zhihu.com/za-js-sdk^  
```
theses don't look like trackers

Reply:
@ryanbr Okay, it has been modified